cmd/minikube/cmd/image.go

```diff
@@ -135,12 +140,14 @@ var loadImageCmd = &cobra.Command{
 		if imgDaemon || imgRemote {
 			image.UseDaemon(imgDaemon)
 			image.UseRemote(imgRemote)
+			out.Step(style.Caching, `Caching and loading images to all profiles.`)
```

Comment:
if we are making out.Step we need to define a step, 
if you wanna use step you can add a Json output
I suggest you either add a json output to minikube image cache command or simply use 

out.Styled instead of out.Step

Reply:
@medyagh thanx I'm  going to use out.Styled and I'm also planning to add progress bar 
https://github.com/cheggaaa/pb

Reply:
@Shivansh-yadav13 are you still working on this ?